bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerFragmentReplicator.java

```diff
@@ -395,14 +395,15 @@ public void readComplete(int rc, LedgerHandle lh,
                 byte[] data = entry.getEntry();
                 final long dataLength = data.length;
                 numEntriesRead.inc();
-                numBytesRead.registerSuccessfulValue(dataLength);
 
                 ByteBufList toSend = lh.getDigestManager()
                         .computeDigestAndPackageForSending(entryId,
                                 lh.getLastAddConfirmed(), entry.getLength(),
                                 Unpooled.wrappedBuffer(data, 0, data.length));
+                int readBytes = toSend.readableBytes();
```

Comment:
why did you modify this? 
`numBytesRead` tracks the data that have been read

Reply:
> why did you modify this?
> `numBytesRead` tracks the data that have been read

yes, this numBytesRead  is equal, I rollback this, thanks @nicoloboschi